framework/core/src/Discussion/Search/FulltextFilter.php

```diff
@@ -42,7 +42,6 @@ public function search(SearchState $state, string $value): void
 
     protected function sqlite(DatabaseSearchState $state, string $value): void
     {
-        /** @var Builder $query */
```

Comment:
These are actually IDE helpers, so that we know exactly which type of Builder (eloquent versus query) is given. Is this now implied from the `$state->getQuery()` instead?

Reply:
That's right, `DatabaseSearchState`'s `getQuery()` method [has a clear return type](https://github.com/flarum/framework/blob/main/framework/core/src/Search/Database/DatabaseSearchState.php#L24-L24) of `Illuminate\Database\Eloquent\Builder` (therefore, it can't be anything else), which is inferred just fine by IDEs, so there is no need to specify it explicitly anymore.